internal/checker/cloudformation/rules/secrets/verify_EC2_user_data_have_no_hard_coded_secrets.rego

```diff
@@ -0,0 +1,50 @@
+# METADATA
+# title: "Verify EC2 User Data have no hard coded secrets."
+# description: "Implement and automate verification processes to ensure EC2 user data does not contain hard-coded secrets, enhancing the security posture of AWS environments."
+# scope: package
+# related_resources:
+# - https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-instance.html
```

Comment:
update this since this is breaking changes
# custom:
#   id: CB_CFT_37
#   severity: High